davinci-grid.md

```diff
@@ -30,24 +30,38 @@ to some special commands provided by `ganga`.
 To create your first `ganga` job type the following:
 
 ```python
-j = Job(application=DaVinci(version='v41r2'))
+j = Job(name='First ganga job')
+myApp = prepareGaudiExec('DaVinci','v41r2', myPath='.')
+j.application = myApp
```

Comment:
I guess to be explicit about the creation of the object. We could say that a one-liner is possible. 

I guess there's nothing one could do with the GaudiExec object that couldn't be done with j.application?